google/validation.go

```diff
@@ -6,6 +6,8 @@ import (
 	"regexp"
 )
 
+const ProjectRegex = "(?:(?:[-a-z0-9]{1,63}\\.)*(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?):)?(?:[0-9]{1,19}|(?:[a-z0-9](?:[-a-z0-9]{0,61}[a-z0-9])?))"
```

Comment:
Oh cool, I hadn't noticed that regex. Do you mind adding a comment of where you got it from? :+1: